pkg/compare/testdata/WhenUsingDiffAllFlag-AllUnmatchedResourcesAppearInSummary/localout.golden

```diff
@@ -1,21 +1,62 @@
-More then one template with same apiVersion, metadata_namespace, kind. These templates wont be used for correlation. To use them use different correlator (manual matching) or remove one of them from the reference. Template names are: apps.v1.DaemonSet.kube-system.kindnet.yaml, apps.v1.DaemonSet.kube-system.kindnet2.yaml
+More then one template with same apiVersion, metadata_namespace, kind. By Default for each Cluster CR that is correlated to one of these templates the template with the least number of diffs will be used. To use a different template for a specific CR specify it in the diff-config (-c flag) Template names are: apps.v1.DaemonSet.kube-system.kindnet.yaml, apps.v1.DaemonSet.kube-system.kindnet2.yaml
+**********************************
+
+Cluster CR: apps/v1_DaemonSet_SomeNS_Name
+Reference File: apps.v1.DaemonSet.kube-system.kindnet2.yaml
+Diff Output: diff -u -N TEMP/apps-v1_daemonset_somens_name TEMP/apps-v1_daemonset_somens_name
+--- TEMP/apps-v1_daemonset_somens_name	DATE
++++ TEMP/apps-v1_daemonset_somens_name	DATE
+@@ -7,4 +7,5 @@
+     app: kindnet
+     k8s-app: kindnet
+     tier: node
++  name: Name
+   namespace: SomeNS
+
+**********************************
+
+Cluster CR: apps/v1_DaemonSet_SomeNS_Test1
+Reference File: apps.v1.DaemonSet.kube-system.kindnet2.yaml
+Diff Output: diff -u -N TEMP/apps-v1_daemonset_somens_test1 TEMP/apps-v1_daemonset_somens_test1
+--- TEMP/apps-v1_daemonset_somens_test1	DATE
++++ TEMP/apps-v1_daemonset_somens_test1	DATE
+@@ -7,4 +7,5 @@
+     app: kindnet
+     k8s-app: kindnet
+     tier: node
++  name: Test1
+   namespace: SomeNS
+
+**********************************
+
+Cluster CR: apps/v1_DaemonSet_SomeNS_Test2
+Reference File: apps.v1.DaemonSet.kube-system.kindnet2.yaml
+Diff Output: diff -u -N TEMP/apps-v1_daemonset_somens_test2 TEMP/apps-v1_daemonset_somens_test2
+--- TEMP/apps-v1_daemonset_somens_test2	DATE
++++ TEMP/apps-v1_daemonset_somens_test2	DATE
+@@ -7,4 +7,5 @@
+     app: kindnet
+     k8s-app: kindnet
+     tier: node
++  name: Test2
+   namespace: SomeNS
+
+**********************************
+
 Summary
-CRs with diffs: 0/0
+CRs with diffs: 3/1
```

Comment:
just  a comment, leaving the test as is because its not relevant to this PR 
but i think this looks funny and we should change it to mkae more sense in case there are multiple CRs from the same template kind

Reply:
Why is that happening if your only matching when you have gotten the match?

Reply:
I intuit that `3/1` means we found 3 CRs that matched a single template?

If that's what this means, I agree it looks weird and maybe is an indication that the message format and/or not what a user would expect.

So let's open that other story, link it here for reference, and go ahead with this change.